cumulus/polkadot-parachain/polkadot-parachain-lib/src/nodes/shell.rs

```diff
@@ -0,0 +1,152 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
```

Comment:
Yeah it is not needed, but it can be an easy follow-up. If not done here, please make an issue and add it to our project board please :) 

It can also be a good one to have the mentor tag and use externals. Seems very straightforward.

Reply:
Opened a new issue for this since it seems a bit separate: https://github.com/paritytech/polkadot-sdk/issues/5898 . I assigned it to myself and I plan to do it immediately after merging this one.